main.go

```diff
@@ -17,26 +19,34 @@ var fileBaseNamesToSkip = []string{".DS_Store"}
 
 // ConfigsModel ...
 type ConfigsModel struct {
-	BuildURL            string
-	APIToken            string
-	IsCompress          string
-	ZipName             string
-	DeployPath          string
-	NotifyUserGroups    string
-	NotifyEmailList     string
-	IsPublicPageEnabled string
+	BuildURL                   string
+	APIToken                   string
+	IsCompress                 string
+	ZipName                    string
+	DeployPath                 string
+	NotifyUserGroups           string
+	NotifyEmailList            string
+	IsPublicPageEnabled        string
+	PublicInstallPageMapFormat string
+}
+
+// PublicInstallPageItem ...
+type PublicInstallPageItem struct {
```

Comment:
I would recommend to name it: `PublicInstallPage` the variable: `publicInstallPage` and it's list: `publicInstallPages`.
Please update the var and input names regarding this (remove the map from everywhere).